superset/assets/javascripts/explore/main.css

```diff
@@ -34,3 +34,32 @@
 .background-transparent {
   background-color: transparent !important;
 }
+
+a {
```

Comment:
Yes the previous solution to get a cursor was to add an `href='#'` prop and this css is better IMO, will move to the theme